principles/architectural.md

```diff
@@ -5,4 +5,22 @@ title:  "Architectual Principles"
 
 # Architectual Practices
 
-Placeholder
+## Introduction
+These pages give advice on the common approaches that we take across the Apprenticeship service. It will detail what approaches should be taken and why.
+
+The approaches and patterns listed are expected to be followed but are not aimed at limiting new components, aspects or approaches. They are given to show the currently preferred approach within the service. Where a new approach or type of implementation is required for a project then it is perfectly acceptable that the approaches listed can be updated. If a new approach is to be taken by a project team then agreement and acceptance within the architecture group needs to be sought and agreed before any design is finalised (No implementation should be carried out without this).  
+
+Our overall approach is driven from the GDS service design manual ([service standard](https://www.gov.uk/service-manual/service-standard)). 
```

Comment:
...umm...interesting one. Not sure. 

As they are currently then yes but 
1. Do we want to be tied to them? 
2. I don't want us to give the impression that we are bound by DfE advice (which this might imply?).

What do you think?

Reply:
I would probably refer to them, but from the perspective of they were considered during the production of the AS ones.  I.e. informed ours, do NOT bind ours.